week-1/K-functions-parameters/exercise3.js

```diff
@@ -1,9 +1,14 @@
-// Write your function here
+// IT'S WORKING BUT I THINK THERE IS A BETTER WAY
+var name = "Daniel";
+
+function createGreeting() {
+  return "Hello, my name is " + name;
+}
 
 var greeting = createGreeting("Daniel");
 
```

Comment:
Same comments apply here as in exercise K-1, make use of function parameters to return a dynamic result.